src/sentry/static/sentry/app/views/eventsV2/miniGraph.tsx

```diff
@@ -76,13 +79,44 @@ class MiniGraph extends React.Component<Props> {
     };
   }
 
+  getChartType({
+    showDaily,
+    yAxis,
+    timeseriesData,
+  }: {
+    showDaily: boolean;
+    yAxis: string;
+    timeseriesData: Series[];
+  }): 'bar' | 'line' | 'area' {
+    if (showDaily) {
+      return 'bar';
+    }
+    if (timeseriesData.length > 1) {
+      switch (aggregateMultiPlotType(yAxis)) {
+        case 'line':
+          return 'line';
+        case 'area':
+          return 'area';
+        default:
+          throw new Error(`Unknown multi plot type for ${yAxis}`);
+      }
+    }
+    return 'area';
+  }
+
   getChartComponent(
-    showDaily: boolean
+    chartType: 'bar' | 'line' | 'area'
```

Comment:
Should this union be a named type? It shows up a few times across our code.

Reply:
Fixed in 3c88c3a